paasta_tools/monitoring/check_k8s_api_performance.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# Copyright 2015-2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import time
+
+from kubernetes import client
+
+from paasta_tools.kubernetes_tools import KubeClient
+
+
+NAMESPACE = "paasta"
+LABEL_SELECTOR = None
+
+
+def parse_k8s_api_performance_options() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+
+    parser.add_argument(
+        "--kube-config-path",
+        "-k",
+        dest="kube_config_path",
+        default=None,
```

Comment:
This should still probably default to the new KUBE_CONFIG_USER_CONF that was added in  https://github.com/Yelp/paasta/pull/3477/files (tl;dr this is the same as your original code, `/etc/kubernetes/paasta.conf`)